docs/tutorials/data-loading/databases.md

```diff
@@ -0,0 +1,144 @@
+# Databases
+
+This tutorial gives an example of how to load data from a database into Qlik Associative Engine using a JDBC connector
+and the gRPC protocol. In this example data resides in mySQL and PostgreSQL.
+
+Several different database connector examples are provided with Qlik Core. See the
+[Data Connector API](../../services/qix-engine/apis/data-loading/introduction.md) section for more information.
+
+## Prerequisites
+
+To follow along in this tutorial, you should have basic understanding of Docker.
+
+You need the following software installed:
+
+* Git
+* Docker
+* docker-compose
+* Node.js
+
+!!! Note
+    Shell commands should be run in a Bash shell.
+    If you are using Windows, we recommend using Git Bash.
+
+## Setup
+
+To run the example code, clone the
+[qlik-oss/core-grpc-jdbc-connector](https://github.com/qlik-oss/core-grpc-jdbc-connector)
+Git repository. Check out the repo documentation to get familiar with the content and structure.
+
+In your shell, make sure current directory is at the repo root.
+
+Set the `ACCEPT_EULA` environment variable as applicable and start the containers by running:
+
+```sh
+ACCEPT_EULA=<yes/no> docker-compose up --build -d
+```
+
+This starts the following services in containers:
+
+* A mySQL database
+* A postgreSQL database
+* A core-grpc-jdbc-connector
+* The Qlik Associative Engine
+
+Verify that all four containers are properly running with:
+
+```sh
+docker ps
+```
+
+## The gRPC JDBC connector
+
+The gRPC JDBC connector runs in its own Docker container and sits between Qlik Associative Engine and the databases
+supporting JDBC connections. It implements the
+[Data Connector API](../../services/qix-engine/apis/data-loading/introduction.md) which makes it possible for the
+engine to communicate over gRPC with it. At the other end it communicates with the PostgreSQL and mySQL containers
+with JDBC connections, in this way working as a data bridge between the engine and the data sources.
+
+The Qlik Associative Engine container must be configured to use the custom JDBC connector by providing some command-line
+options to it. Study the
+[docker-compose.yml](https://github.com/qlik-oss/core-grpc-jdbc-connector/blob/master/example/docker-compose.yml) file
+and notice the following options:
+
+* `-S EnableGrpcCustomConnectors=1` which enables gRPC connectors in the Qlik Associative Engine.
+* `-S GrpcConnectorPlugins="jdbc,jdbc-connector:50051"` which registers a connector of the type `jdbc`, which we tell
+  the Qlik Associative Engine exists on host `jdbc-connector` and listening on port 50051. (`jdbc` is here an arbitrary
```

Comment:
I'm not sure if it's the first part of "jdbc,jdbc-connector:50051" or the second that jdbc is arbitrary.

Reply:
It is the first one before the comma. I'll clarify that :+1: